spring/src/main/java/org/axonframework/spring/messaging/DefaultEventMessageConverter.java

```diff
@@ -60,7 +60,7 @@ public <T> Message<T> convertToOutboundMessage(EventMessage<T> event) {
         headers.put(MESSAGE_TYPE, event.type().toString());
         if (event instanceof DomainEventMessage) {
             headers.put(AGGREGATE_ID, ((DomainEventMessage<?>) event).getAggregateIdentifier());
-            headers.put(AGGREGATE_SEQ, ((DomainEventMessage<?>) event).getSequenceNumber());
+            headers.put(AGGREGATE_SEQ, Long.toString(((DomainEventMessage<?>) event).getSequenceNumber()));
```

Comment:
Why is this change necessary? `headers` is of type `Map<String, Object>`, so it should work out